web/custom.js

```diff
@@ -0,0 +1,80 @@
+const webcam = document.getElementById('webcam');
+const canvas = document.getElementById('canvas');
+const user = document.getElementById('userForm');
+const loader = document.getElementById('heartLoader');
+const context = canvas.getContext('2d');
+const biggerText = document.getElementById('issaBiggerText')
+const smallerText = document.getElementById('issaSmallerText')
+const constraints = {
+  video: true,
+};
+//Enables webcam
+navigator.mediaDevices.getUserMedia(constraints)
+  .then((stream) => {
+    webcam.srcObject = stream;
+  });
+
+// Checks if "^" Key is pressed
+document.addEventListener("keydown", function (e) {
+  if (e.keyCode == 192) {
```

Comment:
Why keyCode = 192 (`)? Just random pick?

Reply:
Yes, the keyCode was a random Pick by me and will be shown in the documentation.